.circleci/config.yml

```diff
@@ -248,54 +272,6 @@ workflows:
     jobs:
       - flake8
       - build
-      - test-other:
-          requires:
-            - build
-      - test-browser:
-          requires:
-            - build
-      - test-ab:
-          requires:
-            - build
-      - test-c:
-          requires:
-            - build
-      - test-d:
-          requires:
-            - build
-      - test-e:
-          requires:
-            - build
-      - test-f:
-          requires:
-            - build
-      - test-ghi:
-          requires:
-            - build
-      - test-jklmno:
-          requires:
-            - build
-      - test-p:
-          requires:
-            - build
-      - test-qrst:
-          requires:
-            - build
-      - test-uvwxyz:
-          requires:
-            - build
-      - test-binaryen0:
-          requires:
-            - build
-      - test-binaryen1:
-          requires:
-            - build
-      - test-binaryen2:
-          requires:
-            - build
-      - test-binaryen3:
-          requires:
-            - build
-      - test-sanity:
+      - test-browser-chrome:
```

Comment:
I assume removing all these is for quicker testing on CI via github PRs

Reply:
Yeah.. just testing.